src/pages/shopPhonePage.js

```diff
@@ -0,0 +1,46 @@
+class ShopPhonePage {
+    brandDropdown = '#select2-chosen-2'
+    appleBrand = '.select2-results > li:nth-child(3)'
+    ramSizeFrom = '#arrCatalogFilter_750_MIN'
+    ramSizeTo = '#arrCatalogFilter_750_MAX'
+    metallMaterialCheckbox = 'label[for*="1086_1914675509"]'
+    year2019Checkbox = 'label[for*="1039_3327493404"]'
+    advancedSearch = 'div.js-accordeon > a > span'
+    showButton = '#set_filter'
+    firstPhone = ':nth-child(1) > * > .catalog_tovar_title'
+
+    selectBrand(brandName) {
+        cy.get(this.brandDropdown).click()
+        cy.get(brandName).trigger('mousemove').click()
+        return this
+    }
+
+    clickAdvancedSearch() {
+        cy.get(this.advancedSearch).click()
+        return this
+    }
+    
+    selectMaterial(metallMaterialCheckbox) {
```

Comment:
You should pass `metallMaterialCheckbox` in parameters in test.
Here, in PO, you should name parameter optional, e.g. `material`.
Also try to avoid `checkbox` in parameter, because current method selects particular material, e.g. `metal`.
If you want to specify `checkbox` - then rename method e.g. `selectMaterialCheckbox.`